src/css/swagger-ui.css

```diff
@@ -1,5 +1,10 @@
+#swagger-ui {
+  margin-bottom: 4em;
+}
+
 .information-container.wrapper,
-.float-right {
+.float-right,
+section.models {
   display: none !important;
```

Comment:
Is there a way of getting rid of this display:none?

Reply:
what do you mean exactly? Swagger comes with generated portions we just dont want to show. Digging into swagger for getting portions not rendered at all kinda sounds too much of effort for simply using display: none;